contributing/documentation/overview.rst

```diff
@@ -107,6 +107,12 @@ your project:
     upstream  https://github.com/symfony/symfony-docs.git (fetch)
     upstream  https://github.com/symfony/symfony-docs.git (push)
 
+Fetch the upstream branch list.
```

Comment:
@chancegarcia You can do that if you like to. But that's not necessary as we also can just do that while merging your PR. :)